src/YieldDistributor.sol

```diff
@@ -23,6 +23,8 @@ import {IYieldDistributor} from "src/interfaces/IYieldDistributor.sol";
 contract YieldDistributor is IYieldDistributor, OwnableUpgradeable {
     /// @notice The address of the $BREAD token contract
     Bread public BREAD;
+    /// @notice The address of the $BUTTEREDBREAD token contract
+    ERC20VotesUpgradeable public BUTTEREDBREAD;
```

Comment:
new storage variables need can't be added in the middle, they need to be appended

Reply:
addressed with bf4f97e and opened #96 to track out of order variables